.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-18.04]
-        java: [8]
+        java: [ 8.0.192, 8 ]
```

Comment:
Why specify a specific version here?

Reply:
> Why specify a specific version here?

Glad you asked! It’s often good practice. Sometimes the latest JDK version can **fail** the tests/build process and the fixed(major) version will **pass** which indicates something in the latest JDK caused the failure and not your code.

Here’s an example:
https://github.com/HdrHistogram/HdrHistogram/blob/master/.github/workflows/maven.yml

More coverage on testing goes a long way.🙂
-Carl

Reply:
- But why `8.0.192` specifically, why not `8.0.292` or  `8.0.302` ?
- What about fixed versions in other PR's like:
    - https://github.com/UweTrottmann/SeriesGuide/pull/827/files
    - https://github.com/marytts/marytts/pull/1060/files

Reply:
Sure you can add those too🙂 . Those 8.0.192 is a known major releases for many organizations. Often there are hardened baselines in IT orgs, where major fixed releases are deployed to production, if you build on latest and don’t test for them you can’t guarantee your library will work in production.

Reply:
I see. I actually haven't heard of an open source library building against a specific JVM patch version, or organisation being stuck on a specific patch version for a very long time. It sounds like a bad thing to encourage if I'm honest, since security updates are important. 

I think those orgs will run their own tests, and I also because of the nature of our project it is very unlikely that our features will break on a different JVM patch version.

Thanks again for creating the discussion though.